packages/bits/src/lib/expander/expander.component.html

```diff
@@ -26,9 +26,9 @@
         </div>
     </div>
     <div class="nui-expander__body"
-         [class.hide-left-border]="hideLeftBorder"
-         [@expandedState]="state"
```

Comment:
I wonder if we can get rid of the `state` property in the component and just get away with a simple boolean value for `open` instead of the complex setter and getter methods for `open` that we currently have.

Reply:
We still need the state and that logic in order to populate the class with expanded or collapsed.

Reply:
okay